Classes/GlobalStateExplorers/Globals/FLEXGlobalsEntry.h

```diff
@@ -81,15 +85,20 @@ typedef void (^FLEXGlobalsEntryRowAction)(__kindof UITableViewController * _Nonn
 @interface FLEXGlobalsEntry : NSObject
 
 @property (nonatomic, readonly, nonnull)  FLEXGlobalsEntryNameFuture entryNameFuture;
+@property (nonatomic, readonly)           UITableViewCellAccessoryType cellAccessoryType;
 @property (nonatomic, readonly, nullable) FLEXGlobalsEntryViewControllerFuture viewControllerFuture;
 @property (nonatomic, readonly, nullable) FLEXGlobalsEntryRowAction rowAction;
 
-+ (instancetype)entryWithEntry:(Class<FLEXGlobalsEntry>)entry row:(FLEXGlobalsRow)row;
++ (instancetype)entryWithEntry:(Class<FLEXGlobalsEntry>)entry
+             cellAccessoryType:(UITableViewCellAccessoryType)cellAccessoryType
+                           row:(FLEXGlobalsRow)row;
 
 + (instancetype)entryWithNameFuture:(FLEXGlobalsEntryNameFuture)nameFuture
+                  cellAccessoryType:(UITableViewCellAccessoryType)cellAccessoryType
                viewControllerFuture:(FLEXGlobalsEntryViewControllerFuture)viewControllerFuture;
 
 + (instancetype)entryWithNameFuture:(FLEXGlobalsEntryNameFuture)nameFuture
+                  cellAccessoryType:(UITableViewCellAccessoryType)cellAccessoryType
                              action:(FLEXGlobalsEntryRowAction)rowSelectedAction;
```

Comment:
In addition to these new ones, can we keep the original signatures around and have them default to the disclosure indicator? Removing them is a breaking change I'd like to avoid